worker/cyclone-worker.go

```diff
@@ -170,11 +172,18 @@ func createVersion(vcsManager *vcs.Manager, event *api.Event) {
 		output.Close()
 		event.Output = nil
 		worker_log.SetWatchLogFileSwitch(output.Name(), false)
+		// wait the send the log to kafka throuth cyclone server totally
```

Comment:
// Wait until log has been pushed into cyclone server.
By the way, why not use waitutil to do this. 🤔